code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/github/resource/GithubClient.java

```diff
@@ -201,7 +207,10 @@ public GithubAccountInfo getGithubUserByName(String username) {
     }
 
     public GithubAccountInfo getGithubOrgByName(String org) {
-        String gitHubAccessToken = (String) ApiContext.getContext().getApiRequest().getAttribute(GithubConstants.GITHUB_ACCESS_TOKEN);
+        String githubAccessToken = (String) ApiContext.getContext().getApiRequest().getAttribute(GithubConstants.GITHUB_ACCESS_TOKEN);
```

Comment:
Better null checks for accessToken so that when it is not present the correct error is presented to the user.